src/Microsoft.AspNetCore.WebHooks.Receivers/Middleware/WebHookReceiverMiddleware.cs

```diff
@@ -57,12 +57,14 @@ public static void ConfigureMapping(IApplicationBuilder app)
         /// <returns></returns>
         private static async Task ProcessWebHook(HttpContext context, string webHookReceiver, string id)
```

Comment:
Question: Should I revise the entire thing to merge the managers into the middleware, Simplifying the entire model?  Basically have Middleware > Receivers > Handlers?  This feels like a much simpler approach and having been through enough now, this may make for a simpler solution.